src/Database/Adapter/Mongo.php

```
@@ -0,0 +1,2215 @@
+    {
+        $id = $this->getNamespace() . '_' . $this->filter($id);
+        return (!!$this->getClient()->dropCollection($id));
+    }
+
+    /**
+     * Analyze a collection updating it's metadata on the database engine
+     *
+     * @param string $collection
+     * @return bool
+     */
+    public function analyzeCollection(string $collection): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$collection'. (Unused Code Rules)

(UnusedFormalParameter)